v4-client-js/src/clients/native.ts

```diff
@@ -461,8 +462,13 @@ export async function faucet(
     }
 
     const response = await faucetClient.fill(wallet.address!, subaccountNumber, amount);
+    const sanitized: Response = {
+      data: response.data,
+      status: response.status,
+      headers: response.headers,
+    };
 
-    return encodeJson(response.data);
+    return encodeJson(sanitized);
```

Comment:
why can't we pass `response` directly into `encodeJson`?

Reply:
"response" is actually an AxiosResponse object that contains additional fields:

```
export interface AxiosResponse<T = any, D = any>  {
  data: T;
  status: number;
  statusText: string;
  headers: RawAxiosResponseHeaders | AxiosResponseHeaders;
  config: AxiosRequestConfig<D>;
  request?: any;
}
```

The "request" field contains circular references which breaks `encodeJson`